### PR TITLE
Ability to change NTP server from config

### PR DESCRIPTION
**Pull request by Baterka**
I added ability to change NTP server used to sync time into config, so you are able to use different server in case you cannot use cloudflare's `time.cloudflare.com`. I personally cannot use it, because my hosting blocks it and not willing to unblock it...
Discussed on https://github.com/GeyserMC/Geyser/issues/2330

### CODE REVIEW

**Comment by Konicai** [comment]
The config version should be left at 4, since no breaking changes are made. If a config file doesn't have the new `ntp-server` key, Geyser will just use the default

File: connector/src/main/resources/config.yml [hunk 1]
```diff
@@ -79,8 +79,8 @@ floodgate-key-file: key.pem
 #    password: javaccountpassword123 # Your Minecraft: Java Edition password
 #    microsoft-account: true # Whether the account is a Mojang or Microsoft account.
 #
-#  bluerkelp2: 
-#    email: not_really_my_email_address_mr_minecrafter53267@gmail.com 
+#  bluerkelp2:
```
Comment (Camotoy) [inline connector/src/main/resources/config.yml:82]: Please revert this diff.

File: connector/src/main/java/org/geysermc/connector/GeyserConnector.java [hunk 2]
```diff
@@ -191,7 +191,7 @@ private GeyserConnector(PlatformType platformType, GeyserBootstrap bootstrap) {
 
         TimeSyncer timeSyncer = null;
         if (config.getRemote().getAuthType() == AuthType.FLOODGATE) {
-            timeSyncer = new TimeSyncer(Constants.NTP_SERVER);
```
Comment (Camotoy) [inline connector/src/main/java/org/geysermc/connector/GeyserConnector.java:194]: You can delete this from the Constants file as well.

File: connector/src/main/resources/config.yml (hunk 1 above)
Comment (Baterka) [reply connector/src/main/resources/config.yml:82]: I see, that I changed that, but why should it have spaces at the end?

Comment (Baterka) [reply connector/src/main/resources/config.yml:82]: IntelliJ IDEA really dont like me doing that... It always removes one of them before commit :D

Comment (rtm516) [reply connector/src/main/resources/config.yml:82]:
```suggestion
#  bluerkelp2:
```

**Comment by rtm516** [comment]
It may be worth changing the default to `0.pool.ntp.org` as its less likely to be blocked.

**Comment by Baterka** [comment]
> It may be worth changing the default to `0.pool.ntp.org` as its less likely to be blocked.

I think it does not matter now, when anyone can change it...

**Comment by Konicai** [comment]
> > It may be worth changing the default to `0.pool.ntp.org` as its less likely to be blocked.
> 
> I think it does not matter now, when anyone can change it...

Defaults are usually suppose to be the best option, not to mention that most people who have issues with CF's ntp server won't know what to do

**Comment by Baterka** [comment]
> > > It may be worth changing the default to `0.pool.ntp.org` as its less likely to be blocked.
> > 
> > 
> > I think it does not matter now, when anyone can change it...
> 
> Defaults are usually suppose to be the best option, not to mention that most people who have issues with CF's ntp server won't know what to do

`time.cloudflare.com` was default until now... That means that maybe someone have it allowed specifically in firewall and this change will break it... I think having default to cloudflare is ok...

**Comment by Baterka** [comment]
Will someone merge this?

**Comment by Camotoy** [comment]
We haven't forgotten about this PR! There are other things holding it up.

**Comment by Camotoy** [comment]
We're just going to remove the NTP feature entirely. Thank you so much for the PR anyway!

**Pull request closed without merging**